web/html/src/branding/css/base/components/dropdown-menu.scss

```diff
@@ -9,7 +9,7 @@
 
   a {
     color: inherit;
-    display: block;
+    display: inline-flex;
```

Comment:
I think I'm missing some context here, was there a bug here, what issue does this change fix?

Reply:
Thanks for the explanation, yeah that looks incorrect in that case, thank you for catching it.  

Regarding the files, `theme.scss` is the successor of the old global theme file, we're trying to cut it out into modules over time, but it takes time as we go bit by bit. For now please keep both since there might be jsp pages that also rely on some of those styles, not only React code.